static/templates/admin_user_group_list.hbs

```diff
@@ -9,19 +9,16 @@
             <img class="checkmark" src="/static/images/checkbox-green.svg" />
             {{t 'Saved' }}
         </button>
-        <button class="button save-status btn-danger small">
-            {{t 'Discard changes' }}
-        </button>
```

Comment:
This seems like an independent change -- is this button no longer used, or something?

Reply:
I think 'Discard' is used to undo the last changes (Ex: for adding the user), but there is no need to do so because we can already change the group members by one click on 'x' (on the right of the person name). That's why i make a decision to eliminate the 'Discard' button, which looks to me more UI upgraded.
![GifMaker_20210318102941823](https://user-images.githubusercontent.com/39272041/111575674-e9303800-87d4-11eb-8573-3b7e5ecd40bf.gif)